src/integration_tests/telemetry_async.cpp

```diff
@@ -18,6 +18,7 @@ static void print_armed(bool armed);
 static void print_quaternion(Telemetry::Quaternion quaternion);
 static void print_euler_angle(Telemetry::EulerAngle euler_angle);
 static void print_angular_velocity_body(Telemetry::AngularVelocityBody angular_velocity_body);
+static void print_vfr_hud(Telemetry::VfrHUD vfr_hud);
```

Comment:
```suggestion
static void print_vfr_hud(Telemetry::VfrHud vfr_hud);
```

VfrHUD -> VfrHud, so that we can parse it (VfrHud becomes vfr_hud, but VfrHUD becomes vfr_h_u_d when we automatically parse it)